src/string.ts

```diff
@@ -1,35 +1,67 @@
 // NOTE: this is copied here rather than imported so that
 // end users don't end up importing the NonEmptyArray module
-// if they only wanted to import the string module.
+// if they only wanted to import the String module.
 interface NonEmptyArray<A> extends ReadonlyArray<A> {
     0: A
 }
 
-/** A boolean check that also serves as a type guard
- * which holds if the given string is the string literal
- * type `""`.
+/**
+ * A boolean check that also serves as a type guard which narrows the
+ * type to the string literal `""`.
+ *
+ * @category Utils
+ * @category Type Guards
+ *
+ * @returns boolean
  */
 const isEmpty = (s: string): s is "" => s === ""
 
-/** A curried version of the built-in trim method. */
+/**
+ * Curried version of the built-in trim method.
+ *
+ * @category Utils
+ */
 const trim = (s: string) => s.trim()
 
-/** A curried version of the built-in toLowerCase method. */
+/**
+ * Curried version of the built-in toLowerCase method.
+ *
+ * @category Utils
+ */
 const toLowerCase = (s: string) => s.toLowerCase()
 
-/** A curried version of the built-in toUpperCase method. */
+/**
+ * Curried version of the built-in toUpperCase method.
+ *
+ * @category Utils
+ */
 const toUpperCase = (s: string) => s.toUpperCase()
 
-/** A type guard that holds true when `u` is a string. */
+/**
+ * Type guard that holds true when `u` is a string.
+ *
+ * @category Type Guards
+ *
+ * @returns boolean
+ */
 const isString = (u: unknown): u is string => typeof u === "string"
 
-/** Returns the length of the string */
+/**
+ * Get the length of a string.
+ *
+ * @category Utils
+ */
 const length = (s: string) => s.length
 
-/** A curried version of the built-in split method that
- * is guaranteed to always return at least one entry. If
- * the split fails to produce at least one entry, the entire
- * input string is returned as a single-element array.
+/**
+ * Curried version of the built-in split method that
+ * is guaranteed to always return at least one entry.
+ *
+ * @remarks
+ * If the split fails to produce at least one entry, the entire
+ * input string is returned as a single- element array.
```

Comment:
```suggestion
 * input string is returned as a single-element array.
```

Assuming this change wasn't intentional?